File: code-coaches/easy/paint-costs/PaintCosts.java
import java.util.Scanner;

public class PaintCosts {

    public static int getTotalPaintCosts(int numColors) {
        double basePrice = 40.0;
        double paintPrice = 5.0;
        double taxPercent = 10.0;

        double subtotal = basePrice + (paintPrice * colorsNumber);
        int total = (int) Math.Ceiling((subtotal / taxPercent) + subtotal);

        return total;
    }

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int numColors = input.nextInt();

		System.out.println(getTotalPaintCosts(numColors));

		input.close();
	}

}

File: code-coaches/easy/paint-costs/paint_costs.py
from math import ceil


def get_total_paint_costs(num_colors: int) -> int:
    base_price = 40.0
    paint_price = 5.0
    tax_percent = 10.0

    subtotal = base_price + (paint_price * num_colors)
    total = ceil((subtotal / tax_percent) + subtotal)

    return total


def main():
    num_colors = int(input())
    print(get_total_paint_costs(num_colors))


if __name__ == "__main__":
    main()
